<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScannerGuiClass</class>
 <widget class="QMainWindow" name="ScannerGuiClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MHY扫码器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditUname">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelUname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前选中用户名：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLiveName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>直播间ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLiveId">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>version</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>343</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>项目源码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;a href=&quot;https://github.com/Theresa-0328/MHY_Scanner&quot;&gt;https://github.com/Theresa-0328/MHY_Scanner</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget 
{
    background:#fbfdfd;
    border:1px solid #bfb1f6;
 
    font-size:13px;                
    font-family:&quot;Microsoft YaHei&quot;;
    color:#37c61a;                
}
 
QTableWidget::item 
{
	border:none;
    border-bottom:1px solid #bfb1f6;
    border-right:1px solid #bfb1f6;
}
 
QTableWidget::item:selected    
{
    color:red;        
    background:#cecbd7;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>BiliBili</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>抖音</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>虎牙</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelLiveName_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>直播平台：</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>100</number>
     </property>
     <property name="leftMargin">
      <number>60</number>
     </property>
     <property name="rightMargin">
      <number>60</number>
     </property>
     <item>
      <widget class="QPushButton" name="pBtstartScreen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>监视屏幕</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtStream">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>监视直播间</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoLogin">
       <property name="text">
        <string>自动二次确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoScreen">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>启动时自动监视屏幕</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoExit">
       <property name="text">
        <string>扫码成功后自动退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>账号管理</string>
    </property>
    <addaction name="action1_3"/>
    <addaction name="action2_3"/>
    <addaction name="action1_4"/>
    <addaction name="action1_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>127</y>
      <width>120</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action1_2"/>
    <addaction name="action2_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="a1">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>反馈问题</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>项目主页</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>帮助和反馈</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>添加账号</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>删除账号</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>设为默认账号</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>打开配置文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/ScannerGui.qrc"/>
 </resources>
 <connections/>
</ui>
